event/tests/test_event_list_ui.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+from ..models import Event
+
+
+@pytest.mark.django_db
+class TestEventList:
+    def test_event_list(self, client, user1):
+        events = Event.manager.all()
+        client.force_login(user1.user)
+        response = client.get(f'/{user1.user.id}/event/event_list/')
+        assert response.status_code == 200
+        for event in events:
+            assert event.name in response.content.decode()
```

Comment:
because the response.content.decode() does not change during the loop 
maybe it will be better to assign it to a variable out of the loop and use it like:
```suggestion
        resp_content = response.content.decode()
        for event in events:
            assert event.name in resp_content
```

Please do the same in all the rest of the tests